vm/src/builtins/dict.rs

```diff
@@ -542,12 +542,13 @@ impl Py<PyDict> {
         key: &K,
         vm: &VirtualMachine,
     ) -> PyResult<PyObjectRef> {
-        if let Some(value) = self.entries.get(vm, key)? {
+        let key = key.to_pyobject(vm);
```

Comment:
could you tell me more what was the motivation of this changes?

Reply:
Okay, it seems this is not intended but accidently added to this branch.
Please clean up the branch to only contain fstring changes and push force it.